.vscodeignore

```diff
@@ -1,6 +1,8 @@
 .vscode/**
 .vscode-test/**
 .gitignore
+docs/**
+media/**
```

Comment:
Uhm, an image is used in the README, which is probably necessary for publishing the package.

Reply:
Yes, we should exclude the "main" screenshot from the ignore list.

Reply:
I'm now keeping the screenshot used in the README.  Unfortunately now the size of the archive is up to 723 KiB, but I guess we can't do much about it, and anyway it isn't _that_ bad either.